docs/using/recipes.md

```diff
@@ -177,17 +175,15 @@ Once built, run using the command:
 docker run -it --rm -p 8888:8888 -p 8787:8787 jupyter/scipy-dasklabextension:latest
 ```
 
-Ref:
-[https://github.com/jupyter/docker-stacks/issues/999](https://github.com/jupyter/docker-stacks/issues/999)
+Ref: <https://github.com/jupyter/docker-stacks/issues/999>
 
 ## Let's Encrypt a Notebook server
 
 See the README for the simple automation here
-[https://github.com/jupyter/docker-stacks/tree/master/examples/make-deploy](https://github.com/jupyter/docker-stacks/tree/master/examples/make-deploy)
+<https://github.com/jupyter/docker-stacks/tree/master/examples/make-deploy>
```

Comment:
Is this correct? I'm not sure how this is parsed, I've never used `<>` associated with links until the markdown formatted introduced in this repo started making them show up.

Reply:
Nevermind i'm quite confident it is. I assume `<>` means "turn this into a link, where the label is the target.

Reply:
Yes, these links are official and work well.
Just a simple syntax for a simple case:
https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses